ls1intum/Apollon
# Fix wrong package size on export

### Checklist

### Description
Fixes #146 

### Steps for Testing
1. Create a class diagram with two packages side by side
2. Exported package should keep its size
3. Try exporting all the other elements to be sure nothing breaks

### Screenshots
#### On editor
<img width="813" alt="Screen Shot 2021-02-26 at 02 07 53" src="https://user-images.githubusercontent.com/16825065/109240472-d6c76d80-77d7-11eb-9167-d60cdb975d94.png">

#### Before

<img width="1087" alt="Screen Shot 2021-02-26 at 02 10 00" src="https://user-images.githubusercontent.com/16825065/109240426-c4e5ca80-77d7-11eb-8edd-24dd057e1fd6.png">

#### After

<img width="766" alt="Screen Shot 2021-02-26 at 02 08 07" src="https://user-images.githubusercontent.com/16825065/109240451-d202b980-77d7-11eb-9481-e22330646ff7.png">
